scripts/setup-centos9.sh

```diff
@@ -208,6 +208,10 @@ function install_arrow {
     -DCMAKE_BUILD_TYPE=Release \
     -DARROW_BUILD_STATIC=ON \
     -DThrift_SOURCE=BUNDLED
```

Comment:
Can we move thrift install to the beginning and skip bundling thrift?

Reply:
Yes, we can. But just letting arrow build the required thrift can reduce maintenance code in velox. Then, we don't need to care about thrift build (e.g., which version or which build options to use) when upgrading arrow in the future. Does it make sense?

Reply:
Makes sense!